src/main/kotlin/dev/ocpd/jsensible/internal/nullability/NullabilityAnnotations.kt

```diff
@@ -10,19 +12,19 @@ import dev.ocpd.jsensible.internal.JavaClassPredicates.anyOf
  */
 internal object NullabilityAnnotations {
 
-    internal fun nullableAnnotations(): DescribedPredicate<JavaAnnotation<*>> = rawType(
-        anyOf(
-            "org.jetbrains.annotations.Nullable",
-            "javax.annotation.Nullable",
-            "org.springframework.lang.Nullable"
-        )
-    ).forSubtype()
-
     internal fun nonNullAnnotations(): DescribedPredicate<JavaAnnotation<*>> = rawType(
         anyOf(
             "org.jetbrains.annotations.NotNull",
             "javax.annotation.Nonnull",
             "org.springframework.lang.NonNull"
         )
     ).forSubtype()
+
+    internal fun nullableAnnotations(): DescribedPredicate<JavaClass> =
+        simpleName("Nullable")
+
+    internal fun springNullableAnnotation(): DescribedPredicate<JavaClass> =
```

Comment:
Just curious. It's necessary to wrap this into a `anyOf()`?

Reply:
After my research, I found that it is not necessary and the modification has been tested without problems.